tox.ini

```diff
@@ -54,6 +54,9 @@ commands =
     sphinx-build -b html -d build/doctrees source build/html
 
 [testenv:virtualenv_run-dev]
+deps =
+    -rrequirements.txt
+    -rrequirements-dev.txt
```

Comment:
this is unrelated, but it was annoying not having things like pre-commit available in the dev venv